base_tier_validation/models/tier_validation.py

```diff
@@ -435,7 +435,7 @@ def _notify_review_requested(self, tier_reviews):
         subscribe = "message_subscribe"
         post = "message_post"
         if hasattr(self, post) and hasattr(self, subscribe):
-            for rec in self:
+            for rec in self.sudo():
```

Comment:
why?

Reply:
Oh, You already posted that info here https://github.com/OCA/server-ux/pull/747#issuecomment-1793072646.

Reply:
I am not sure that fixing the tests is enough reason to add that sudo... could you find another solution?

Reply:
Well, all other message posts have been refactored to use sudo.
This one is the left over.. 

Idk how to solve differently. Any suggestions?

Reply:
I see, thanks for pointing that out